doc/man3/EVP_PKEY_size.pod

```diff
@@ -0,0 +1,39 @@
+=pod
+
+=head1 NAME
+
+EVP_PKEY_size, EVP_PKEY_bits, EVP_PKEY_security_bits
+- EVP_PKEY information functions
+
+=head1 SYNOPSIS
+
+ #include <openssl/evp.h>
+
+ int EVP_PKEY_size(const EVP_PKEY *pkey);
+ int EVP_PKEY_bits(const EVP_PKEY *pkey);
+ int EVP_PKEY_security_bits(const EVP_PKEY *pkey);
+
+=head1 DESCRIPTION
+
+EVP_PKEY_size() returns the size of the key in I<pkey> in bytes.
+
+EVP_PKEY_bits() returns the size of the key in I<pkey> in bits.
+
+EVP_PKEY_security_bits() returns the number of security bits of the given
+I<pkey>, bits of security is defined in NIST SP800-57.
+
+=head1 RETURN VALUES
+
+EVP_PKEY_size(), EVP_PKEY_bits() and EVP_PKEY_security_bits() return a
+positive number, or 0 if this size isn't available.
+
+=head1 COPYRIGHT
+
+Copyright 2000-2018 The OpenSSL Project Authors. All Rights Reserved.
```

Comment:
Maybe update copyright?

Reply:
@romen took so much space that I totally glossed over this little comment. Thanks for the ping.
Yeah, copyright years will be changed.

Reply:
I am the worst! Sorry @slontis 😂

Reply:
Fixed

Reply:
Futzing with copyright is also done when a release is made so I wouldn't lose sleep over it.

Reply:
Except this is a *new* file, so it should only say 2020. Our scripts don't futz with the starting year

Reply:
Either way, it's fixed already, so...